skyportal/tests/frontend/test_groups.py

```diff
@@ -77,3 +80,20 @@ def test_delete_group_user(driver, super_admin_user, user, public_group):
     time.sleep(0.5)
     assert len(driver.find_elements_by_xpath(
         f'//a[contains(.,"{user.username}")]')) == 0
+
+
+def test_token_user_update_group(driver, super_admin_user, public_group):
+    auth_token = create_token(public_group.id, ['Manage groups'])
+    response = requests.put(
```

Comment:
There are now enough of these calls that we may want to start factoring them out.  But that can wait for another PR.